py_swf/clients/workflow.py

```diff
@@ -2,8 +2,21 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+from collections import namedtuple
+
 __all__ = ['WorkflowClient']
 
+CountWorkflowsResult = namedtuple(
```

Comment:
nit: I'd also add a link to the response syntax in the SWF APIs (something similar to :meth:`~SWF.Client.count...`.)